qtgui/resources/ApplicationWindow.qml

```diff
@@ -0,0 +1,76 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import starturtle.oneBit 1.0
+
+ApplicationWindow {
+    id: mainWindow
+    visible: true
+    width: 800
+    height: 600
+    title: qsTr("Image Pixelation")
+
+    menuBar: MenuBar {
+        Menu {
+            title: qsTr("File")
+            MenuItem {
+                text: qsTr("&Load")
+                onTriggered: {
+					imagePreview.getInputFile()
+				}
+            }
+            MenuItem {
+                text: qsTr("Select Out&path")
+                onTriggered: {
+					imagePreview.getOutputFile()
+				}
+            }
+            MenuItem {
+                text: qsTr("Exit")
+                onTriggered: Qt.quit();
+            }
+        }
+		Menu {
+			title: qsTr("&Pixelation")
+			MenuItem {
+				text: qsTr("&Run")

```

Comment:
it would be much nicer if this went automagically on Load with the default arguments. But for that to work with the current setup, one would need a default output name.